net/rpki-client/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                rpki-client
+version             7.1
+categories          net
+platforms           darwin
+license             ISC
+maintainers         nomaintainer
+description         A Resource Public Key client
+long_description    rpki-client is a FREE, easy-to-use implementation \
+                    of the Resource Public Key Infrastructure (RPKI) for \
+                    Relying Parties (RP) to facilitate validation of the \
+                    Route Origin of a BGP announcement.
+homepage            https://www.rpki-client.org/index.html
+master_sites        https://cdn.openbsd.org/pub/OpenBSD/rpki-client/ \
+                    https://cloudflare.cdn.openbsd.org/pub/OpenBSD/
+checksums           rmd160  e67746b08c1e9dad0cf68ebb71eb6ca6f225f16f\
+                    sha256  41140d942d1492dc922e8d70703b30235cbe75cdb0c4de16cccde6c60fcba1ff
+
+depends_lib-append  lib:libtls:libressl
```

Comment:
If this port requires libressl, write the dependency `port:libressl`. This would be unusual and would make it unable to be installed at the same time as any port that uses openssl (which is a lot of ports). If the port can use either openssl or libressl, write the dependency `path:lib/libssl.dylib:openssl`. This is what most ports do. Don't use the `lib:` dependency type because it would allow a copy of that library that is installed outside of MacPorts to satisfy the dependency, and we don't want that in most cases.